source/declarations.tex

```diff
@@ -1770,10 +1770,11 @@
 from an initializer.
 
 \pnum
-The type of a \grammarterm{parameter-declaration} of a
-function declaration\iref{dcl.fct},
+The type of a \grammarterm{parameter-declaration} of
+any of the following three:
```

Comment:
This isn't an improvement.

I think just adding bullets for the three options is good enough to disambiguate.

Reply:
My opinion is that the proposal with column ":" followed by a list, was anyway able to disambiguate.

However, bullets are for sure a good way, let me amend the commit.